dask/dataframe/core.py

```diff
@@ -1266,6 +1266,30 @@ def _bind_operator_method(cls, name, op):
         """ bind operator method like DataFrame.add to this class """
         raise NotImplementedError
 
+    def nunique_approx(self, p=16):
+        """Approximate number of unique rows.
+
+        This method uses the HyperLogLog algorithm for cardinality
+        estimation to compute the approximate number of rows.
+
+        Parameters
+        ----------
+        p : integer, default 16
+            The number of bits of the hash to use for approximating
+            the number of unique elements. (This is the parameter
+            'p' of the HyperLogLog algorithm.)
```

Comment:
Mind adding in a blurb about what increasing/decreasing this number does in simplistic terms?